benchmark/benchmark_umf.hpp

```diff
@@ -45,8 +45,8 @@ struct provider_interface {
         if (state.thread_index() != 0) {
             return;
         }
-        umfCtlExec("umf.provider.by_handle.stats.peak_memory.reset", provider,
-                   NULL, 0);
+        umfCtlExec("umf.provider.by_handle.{}.stats.peak_memory.reset", NULL, 0,
```

Comment:
I wonder if arg + size can be part of va args?

Reply:
nope